src/components/Menu/ContextMenu.tsx

```diff
@@ -88,7 +88,7 @@ export const ContextMenu: FC<Props> = ({
 
   // Normally, the menu takes the form of a floating box. But on Android and
   // iOS, the menu should morph into a drawer
-  const platform = getPlatform();
+  const platform = useMemo(() => getPlatform(), []);
```

Comment:
given this pattern is used in multiple places, it seems like a `usePlatform` hook would be nice